src/Doctrine/Phpcr/PrefixInterface.php

```diff
@@ -22,11 +22,11 @@ interface PrefixInterface
     /**
      * @return string the full path of this document in the repository
      */
-    public function getId();
+    public function getId(): string;
```

Comment:
changing the signature of interface methods is a BC break.

i agree that we should eventually do that, but if we want to quickly release symfony 6 support, we should do that as a minor version without BC breaks.